src/Stratis.Bitcoin.Features.MemoryPool.Tests/FullNodeBuilderTest.cs

```diff
@@ -25,8 +25,8 @@ public void CanHaveAllFullnodeServicesTest()
                 $"-datadir=Stratis.Bitcoin.Features.MemoryPool.Tests/TestData/FullNodeBuilderTest/CanHaveAllServicesTest" });
             var fullNodeBuilder = new FullNodeBuilder(nodeSettings);
             IFullNode fullNode = fullNodeBuilder
-                .UsePowConsensus()
                 .UseBlockStore()
+                .UsePowConsensus()
```

Comment:
Why the order switch?

Reply:
>Why the order switch?

BlockStore.cs: 114
// Throw if block store was initialized after the consensus.
// This is needed in order to rewind consensus in case it is initialized ahead of the block store.